app/router.ts

```diff
@@ -94,8 +94,11 @@ Router.map(function () {
 
   // TODO: Add dark mode support
   this.route('vote', function () {
-    this.route('course-ideas', { path: '/challenge-ideas' });
-    this.route('course-extension-ideas', { path: '/challenge-extension-ideas' });
+    this.route('course-ideas', { path: '/challenges' });
+    this.route('course-extension-ideas', { path: '/challenge-extensions' });
+    // Add redirects for old paths
+    this.route('redirect-challenge-ideas', { path: '/challenge-ideas' });
```

Comment:
@Arpan-206 this is easier to place in [vercel.json](https://github.com/codecrafters-io/frontend/blob/main/vercel.json), doesn't need a separate route definition :)